packages/amplify_authenticator/lib/amplify_authenticator.dart

```diff
@@ -575,17 +574,24 @@ class _AuthenticatorState extends State<Authenticator> {
     _infoSub.cancel();
     _successSub.cancel();
     _stateMachineBloc.close();
-    _hubSubscription?.cancel();
     super.dispose();
   }
 
   Future<void> _waitForConfiguration() async {
-    var config = await Amplify.asyncConfig;
-    setState(() {
-      _config = config;
-      _configInitialized = true;
-      _missingConfigValues = missingConfigValues(config);
-    });
+    try {
+      final config = await Amplify.asyncConfig;
+      setState(() {
+        _config = config;
+        _configInitialized = true;
+        _missingConfigValues = missingConfigValues(config);
+      });
+    } on Exception catch (e) {
+      _showExceptionBanner(
+        type: StatusType.error,
+        message: AuthenticatorException(e).message,
+      );
+      await _waitForConfiguration();
```

Comment:
questions: 1) Are we sure we want to catch all exceptions here or are there specific exceptions we would expect here? 2) Could this get into a loop if the same exception gets thrown again on 2nd call?

Reply:
1) Yes, since we cannot continue without the config object.
2) No, since the only code paths which throw an exception also reset `asyncConfig`